pandas/tests/io/test_sql.py

```diff
@@ -2663,23 +2669,17 @@ def clean_up(test_table_to_drop):
             self.drop_table(test_table_to_drop)
 
         # test if invalid value for if_exists raises appropriate error
-        pytest.raises(ValueError,
-                      sql.to_sql,
-                      frame=df_if_exists_1,
-                      con=self.conn,
-                      name=table_name,
-                      if_exists='notvalidvalue')
+        with pytest.raises(ValueError, match="<insert message here>"):
```

Comment:
dummy message for unconditionally skipped test. xref #20536

Reply:
Yep, that makes sense.